integration-tests/tests/src/tests/dictionary.ts

```diff
@@ -167,6 +167,28 @@ describe("Dictionary", () => {
       expect(Object.keys(item.dict).length).deep.equals(3);
     });
 
+    it("throws when using Symbol as key", function (this: RealmContext) {
+      const sym = Symbol("testSymbol");
+
+      this.realm.write(() => {
+        const item = this.realm.create<Item>("Item", {});
+
+        expect(() => {
+          //@ts-expect-error We need to verify using a symbol as a key throws
```

Comment:
Not a huge deal but I think it would be fine for ts-expect-error messages to be formulated more toward the specific ts issue and less so to surrounding logic (the test name in this case also describes the purpose). At least this is what I've noticed in the rest of the code base 👍 
```suggestion
          //@ts-expect-error Testing invalid key.
```

If changing this, there's 1 more occurrence further below.